src/api/users.py

```diff
@@ -0,0 +1,24 @@
+"""users endpoint module."""
+
+from typing import Any
+
+from flask_restx import Namespace, Resource
+
+from src.users.models import User
+
+
+api = Namespace("users", description="Get users id and name")
+
+
+@api.route("/users/<int:user_id>")
+class UserInfo(Resource):  # type: ignore
+    """Get user's id and name."""
+
+    def get(self, user_id: int) -> dict[str, Any] | None:  # pylint: disable=no-self-use
```

Comment:
Can we make this method to be static and remove `# pylint: disable=no-self-use` ?